docs/docs/reference/contextual/derivation.md

```diff
@@ -284,16 +285,30 @@ object Eq {
         }
     }
 
-  inline given derived[T](given m: Mirror.Of[T]): Eq[T] = {
-    val elemInstances = summonAll[m.MirroredElemTypes]
-    inline m match {
-      case s: Mirror.SumOf[T]     => eqSum(s, elemInstances)
-      case p: Mirror.ProductOf[T] => eqProduct(p, elemInstances)
+  inline def derived[T]: Eq[T] =
+    summonFrom {
+      case m: Mirror.Of[T] =>
+        val elemInstances = summonAll[m.MirroredElemTypes]
+        inline m match {
+          case s: Mirror.SumOf[T]     => eqSum(s, elemInstances)
+          case p: Mirror.ProductOf[T] => eqProduct(p, elemInstances)
+        }
     }
-  }
 }
 ```
 
+> Note that we can explicitly state the dependence on `Mirror[T]` in `derived` by using the
+> following encoding:
+> ```scala
+> inline def derived[T](given m: Mirror.Of[T]): Eq[T]
+> ```
+> This is also equivalent to using the Implicit Function Type:
+> ```scala
+> inline given derived[T]: (m: Mirror.Of[T]) => Eq[T]
```

Comment:
Maybe you want to write `(given m: Mirror.Of[T]) => Eq[T]` for implicit function types.

Reply:
Yeah, good thought. I didn't think of that.